components/Habit.tsx

```diff
@@ -94,20 +120,22 @@ const Habit = ({ entries, habit, streak }: HabitProps) => {
         </span>
       </div>
       {habit.target.quantity === 1 ? (
-        <Image
-          src={
-            todayEntries.length >= 1
-              ? '/images/habit-complete.svg'
-              : '/images/habit-incomplete.svg'
-          }
-          alt={
-            todayEntries.length >= 1
-              ? t('habit:complete')
-              : t('habit:incomplete')
-          }
-          width='32'
-          height='32'
-        />
+        <Button stylesProp={styles.completeButton} onClick={toggleComplete}>
+          <Image
+            src={
+              todayEntries.length >= 1
+                ? '/images/habit-complete.svg'
+                : '/images/habit-incomplete.svg'
+            }
+            alt={
+              todayEntries.length >= 1
+                ? t('habit:complete')
+                : t('habit:incomplete')
+            }
+            width='32'
+            height='32'
+          />
```

Comment:
This sort of thing feels like a good candidate for a component _at some point_

Reply:
Done this in PR #63.